Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM python:3.7-slim
```

Comment:
I think it should be a fully qualified name


```
FROM python:3.7-slim
```
=>

```
FROM docker.io/library/python:3.7-slim
```
(If I remember correctly)